vnext/Microsoft.ReactNative/Modules/NativeUIManager.cpp

```diff
@@ -824,6 +902,12 @@ void NativeUIManager::AddView(ShadowNode &parentShadowNode, ShadowNode &childSha
     }
 
     YGNodeInsertChild(yogaNodeToManage, yogaNodeToAdd, static_cast<uint32_t>(index));
+
+    if (parentNode.m_appliedLayoutConformance) {
```

Comment:
Make a comment that this could be further optimized by unsetting m_appliedLayoutConformance when no ancestors have set values, but that would require an extra tree walk, so it's fine to just assume that once you've used the prop, you always have to do the extra recursion step.